File: Console Apps/LinkedListNodes/Node.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LinkedListNodes
{
    internal class Node
    {
        private int Data;
        private Node Next = null;

        public int DATA { set; get; }
        public Node NEXT { set; get; }

        public Node() { }
        public Node(int _data, Node _next)
        {
            this.Data = _data;
            this.Next = _next;
        }
    }
}

File: HighSchool/Sem4DataProjects/src/A1/Node.java
package A1;

public class Node {
	
	int data;
	//Head
	Node next;
	//Trailer
	Node prev;
	
	//Constructor
	public Node(int data) {
		this.data = data;
		this.next = null;
		this.prev = null;
	}

}
